src/index.ts

```diff
@@ -392,6 +398,9 @@ export const SplunkRum: SplunkOtelWebType = {
     });
 
     this.provider = provider;
+    // TODO: this is so that integration tests can access these APIs, should find a better way
```

Comment:
open question here: what's the right way to allow the integration tests to call the `trace` and `context` APIs?

Reply:
Hmm yeah doesn't really have a good way, I'd say grabbing otel object directly from `window[Symbol.for('opentelemetry.js.api.1')]` is good enough for tests

Reply:
for some reason `window[Symbol.for('opentelemetry.js.api.1')].trace` doesn't seem to have a `setSpan` function. will have to look into that a little more...

Reply:
Yeaah that isn't directly on tracing api but rather uses context (and even then the source for setSpan is in context-utils as a standalone function so it's not really set anywhere)

```js
document.body.addEventListener('click', () => {
  console.log('inspan?', window[Symbol.for('opentelemetry.js.api.1')].context.active().getValue(Symbol.for('OpenTelemetry Context Key SPAN')));
  // setting would be .setValue(Symbol.for(...), span)
});
```

might have to consider creating a separate bundle that exposes otel/api for tests...